backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -1059,6 +1060,29 @@ void convertToAoeObservation_pregnancyStatus_matchesJson() throws IOException {
     JSONAssert.assertEquals(expectedSerialized, actualSerialized, true);
   }
 
+  @Test
+  void convertToAoeObservation_genderofsexualpartners_matchesJson() throws IOException {
```

Comment:
nit: could we camel case `genderofsexualpartners` --> `genderOfSexualPartners`?

Reply:
updated